packages/@aws-cdk/aws-cloudtrail/lib/cloudtrail.ts

```diff
@@ -285,6 +296,7 @@ export class Trail extends Resource {
       cloudWatchLogsRoleArn: logsRole?.roleArn,
       snsTopicName: this.topic?.topicName,
       eventSelectors: this.eventSelectors,
+      isOrganizationTrail: props.isOrganizationTrail ?? false,
```

Comment:
What's the behavior here if the account isn't the owner of an organization and this gets set to true? Does it just fail silently? If so, do we have a way to validate that the account is actually the owner of an organization?